### DISCUSSION TITLE
bromite Crashes on all sites after while

### DISCUSSION THREAD

**Discussion by Haluzer** [bromite/bromite#2656] [Builds]
Hello!
So i just recently updated to last version 103 one from version 50 something.
It looked all really nice and  smply fantastic until i got this white screens coming every single minute no matter what site i went on.
it says "something went wrong"
i tried reinstalling 2 times no result i even installe bromite webview without success. 
Phone info:
Oneplus 10 Por
Model Number: NE2213
Os: oxygen 13.1

**Comment by markeerstift** [comment]
Just use Cromite my man. Bromite has been dead for about 2 years now unfortunately.
